portfolio-edit/add-new-project.php

```diff
@@ -6,14 +6,13 @@
 $sql = "INSERT INTO `portfolio` (`name`, `url`, `comment`) VALUES (:projectName, :projectUrl, :projectComment);";
 
 
-    if (!empty($_POST['pname']) && !empty($_POST['url']) && !empty($_POST['comment'])) {
-        $query = $db->prepare($sql);
-        $query->execute([
-            ':projectName' => $_POST['pname'],
-            ':projectUrl' => $_POST['url'],
-            ':projectComment' => $_POST['comment']
-        ]);
-    } else  {
-        echo 'error';
-
-    }
+if (!empty($_POST['pname']) && !empty($_POST['url']) && !empty($_POST['comment'])) {
+    $query = $db->prepare($sql);
+    $query->execute([
+        ':projectName' => $_POST['pname'],
+        ':projectUrl' => $_POST['url'],
+        ':projectComment' => $_POST['comment']
+    ]);
```

Comment:
once the project has been added, you should redirect back to the admin panel.
something like:
```
$result = $query->execute();
if ($result) {
    header('Location: admin-panel.php');
} else {
    echo 'error';
}
```

Reply:
coool